pokemon_search/src/index.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import reportWebVitals from "./reportWebVitals";
+import Details from "./routes/details";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="details" element={<Details />}>
```

Comment:
This means going to `/details` will also display the `Details` page. What happens when we do that?

Reply:
I guess I was expecting to display the details page when going to /details, but now I've realised that it's not necessary because each details page will differ depending on the pokemon id, e.g. http://localhost:3000/details/1

Would this be a better approach? 

     <Routes>
        <Route path="/" element={<App />} />
        <Route path="details">
          <Route path=":pokemonId" element={<Details />} />
        </Route>
      </Routes>